sdk/pack/variables/variables.go

```diff
@@ -78,7 +78,13 @@ func (v *Variable) Equal(ivp *Variable) bool {
 
 func (v *Variable) AsOverrideString(pID pack.ID) string {
 	var out strings.Builder
-	out.WriteString(fmt.Sprintf(`# variable "%s"`, v.Name))
+
+	fqvn := strings.Join([]string{pID.String(), v.Name.String()}, ".")
+	b := strings.Split(fqvn, ".")
+	b = b[1:]
+	rvn := strings.Join(b, ".")
```

Comment:
```suggestion
	_, rvn, _ := strings.Cut(fqvn, ".")
```

gotta love the [`strings.Cut`](https://pkg.go.dev/strings#Cut)!